aea/skills/behaviours.py

```diff
@@ -71,7 +71,7 @@ def number_of_executions(self) -> int:
     def act_wrapper(self) -> None:
         """Wrap the call of the action. This method must be called only by the framework."""
         if not self.is_done():
-            self.act()
+            super().act_wrapper()
```

Comment:
note, bug... we didn't call the act_wrapper, which was mostly inconsequential before, but no longer is so.